bomb mesh is now a child object so animators can pose it in-engine (temp animation there too)

BenicioLadines opened the pull request.

BenicioLadines committed:
bomb mesh is now a child object so animators can pose it in-engine

TheStress reviewed (commented):
You still didnt implement the model, check the description of the issue I assigned for this task #228

BenicioLadines committed:
Merge branch 'develop' into bomb-enemy-anim

BenicioLadines committed:
imported model for bomb enemy

bomb enemy model is in, but it doesn't quite fit the animation i gave it, do we want a different one?

BenicioLadines committed:
added bomb demon texture

it doesnt work tho.. u_U

BenicioLadines committed:
Model and Texture are in hooray!

BenicioLadines committed:
BOMB IS FINALLY IN

with the animations BABY YEAAAHAHA

TheStress commented:
Requested Changes:
- [x] have the bomb enemy stand still when its not moving (you can make a quick animation clip of the model not moving)
- [x] I think we can have a quick visual for when the bomb enemy is dead, it can be something like going limp or darken the mesh

BenicioLadines committed:
animations with logic are in lmk if theyre good!

TheStress committed:
saved model animator

TheStress committed:
Merge branch 'develop' into bomb-enemy-anim

TheStress committed:
fixed energy blasted bug

TheStress committed:
applied changes to prefab
